xml/ftp_banners.xml

```diff
@@ -1024,16 +1024,17 @@ more text</example>
   </fingerprint>
   <fingerprint pattern="^Welcome to the (?:Cisco )?(?:TelePresence) ([a-zA-Z\s]*?) ((?:MSE )?\d+), version (\d+.\d+\(\d+.\d+\)).*?" flags="REG_ICASE">
     <description>Cisco TelePresence</description>
-    <example os.device="AM GW" os.version="1.1(1.34)" hw.series="3610">Welcome to the Cisco TelePresence AM GW 3610, version 1.1(1.34) </example>
-    <example os.device="IP GW" os.version="2.0(3.32)" hw.series="3510">Welcome to the Cisco TelePresence IP GW 3510, version 2.0(3.32) </example>
-    <example os.device="MCU" os.version="4.3(2.18)" hw.series="4210">Welcome to the Cisco TelePresence MCU 4210, version 4.3(2.18) </example>
-    <example os.device="MCU" os.version="4.3(1.68)" hw.series="MSE 8420">Welcome to the Cisco TelePresence MCU MSE 8420, version 4.3(1.68) </example>
-    <example os.device="Server" os.version="2.1(1.18)" hw.series="7010">Welcome to the Telepresence Server 7010, version 2.1(1.18) </example>
+    <example hw.series="AM GW" os.version="1.1(1.34)" hw.model="3610">Welcome to the Cisco TelePresence AM GW 3610, version 1.1(1.34) </example>
+    <example hw.series="IP GW" os.version="2.0(3.32)" hw.model="3510">Welcome to the Cisco TelePresence IP GW 3510, version 2.0(3.32) </example>
+    <example hw.series="MCU" os.version="4.3(2.18)" hw.model="4210">Welcome to the Cisco TelePresence MCU 4210, version 4.3(2.18) </example>
+    <example hw.series="MCU" os.version="4.3(1.68)" hw.model="MSE 8420">Welcome to the Cisco TelePresence MCU MSE 8420, version 4.3(1.68) </example>
+    <example hw.series="Server" os.version="2.1(1.18)" hw.model="7010">Welcome to the Telepresence Server 7010, version 2.1(1.18) </example>
     <param pos="0" name="os.vendor" value="Cisco"/>
     <param pos="0" name="os.product" value="TelePresence"/>
-    <param pos="1" name="os.device"/>
+    <param pos="0" name="os.device" value="Telecom"/>
```

Comment:
We have a `Video Conferencing`  `hw.device` in our data sets. Would this be a good fit for `os.device` and hw.device` here?

Reply:
Oh yes, thats better.